.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```

Comment:
Tip voor volgende keer:  Voeg aan je gitignore je /.idea toe voor je je gitrepository linkt aan je project in intellij. Dan wordt je .idea map niet mee gepushed